container_images/daisy-builder/main.sh

```diff
@@ -20,6 +20,20 @@ ZONE="$2"
 WORKFLOW_FILE="$3"      # Workflow to run
 GCS_OUTPUT_BUCKET="$4"  # Destination for artifacts
 
+function GENERATE_NEW_VERSION() {
+  VERSION_GENERATE_CMD="/versiongenerator --token-file-path=${GITHUB_ACCESS_TOKEN} --org=${REPO_OWNER} --repo=${REPO_NAME}"
+  VERSION_GENERATE_OUT=$(${VERSION_GENERATE_CMD})
+
+  if [[ $? -ne 0 ]]; then
+    echo -e "could not generate version: $VERSION_GENERATE_OUT"
+  fi
+
+  pattern="Generated version: "
+  local BUILD_ID=$(echo $VERSION_GENERATE_OUT | sed "s/$pattern//")
```

Comment:
can we remove the $pattern from generator command, or add flag to remove it? we control this stuff so we shouldn't have to be doing string parsing on what our utilities provide us

Reply:
done